### PR TITLE
Html pages

### PR DESCRIPTION
**Pull request by mainul-codemen**

### CODE REVIEW

File: learn-in-codemen/04-CSS-Layout/challange_2.css [hunk 1]
```diff
@@ -0,0 +1,159 @@
+* {
```
Comment (maainul) [inline learn-in-codemen/04-CSS-Layout/challange_2.css:1]: Why you are using this

Comment (mainul-codemen) [reply learn-in-codemen/04-CSS-Layout/challange_2.css:1]: for give universal padding and margin

File: learn-in-codemen/04-CSS-Layout/challange_2.css [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
```
Comment (maainul) [inline learn-in-codemen/04-CSS-Layout/challange_2.css:7]: what is body

File: learn-in-codemen/04-CSS-Layout/challange_2.css [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: bold;
+  letter-spacing: 2px;
+  padding: 7px 15px;
+
+  position: absolute;
+  top: -17px;
+  left: -38px;
+
```
Comment (maainul) [inline learn-in-codemen/04-CSS-Layout/challange_2.css:63]: Why this

File: learn-in-codemen/04-CSS-Layout/challange_2.css (hunk 2 above)
Comment (mainul-codemen) [reply learn-in-codemen/04-CSS-Layout/challange_2.css:7]: Screenshot from 2021-08-06 14-46-00

**Pull request merged**